public/admin/config.yml

```diff
@@ -4,7 +4,7 @@
 backend:
   name: github
   repo: covid-projections/covid-projections
-  branch: cms-content-entry
+  branch: faihegberg/update-recommendations-section
```

Comment:
This will cause the tests to fail for now, but I'm just keeping this here for testing purposes before PR gets approved.